### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by io12**

### CODE REVIEW

File: Ghidra/Features/Base/src/main/help/help/topics/ExporterPlugin/exporter.htm [hunk 1]
```diff
@@ -147,7 +147,7 @@ <H4>Advanced</H4>
                     <TD><I><B>Comment Prefix</B></I><BR>
                     </TD>
 
-                    <TD valign="top">the string to append on the begining of comments<BR>
+                    <TD valign="top">the string to append on the beginning of comments<BR>
```
Comment (dragonmacher) [inline Ghidra/Features/Base/src/main/help/help/topics/ExporterPlugin/exporter.htm]: This line would read better as:
`the string to prepend to the beginning of comments`

File: GPL/DemanglerGnu/src/demangler_gnu/c/cp-demangle.c [hunk 2]
```diff
@@ -93,7 +93,7 @@ This file was changed on July 4th, 2019.
 
    IN_GLIBCPP_V3
       If defined, this file defines only __cxa_demangle() and
-      __gcclibcxx_demangle_callback(), and no other publically visible
+      __gcclibcxx_demangle_callback(), and no other publicly visible
```
Comment (dragonmacher) [inline GPL/DemanglerGnu/src/demangler_gnu/c/cp-demangle.c]: This is an open source file we have copied into our project.  Our goal is to change this file (and the others in this directory) as little as possible.   This change needs to be removed.

File: GPL/DemanglerGnu/src/demangler_gnu/c/cplus-dem.c [hunk 3]
```diff
@@ -2651,7 +2651,7 @@ iterate_demangle_function (struct work_stuff *work, const char **mangled,
   /* Iterate over occurrences of __, allowing names and types to have a
      "__" sequence in them.  We must start with the first (not the last)
      occurrence, since "__" most often occur between independent mangled
-     parts, hence starting at the last occurence inside a signature
+     parts, hence starting at the last occurrence inside a signature
```
Comment (dragonmacher) [inline GPL/DemanglerGnu/src/demangler_gnu/c/cplus-dem.c]: This is an open source file we have copied into our project.  Our goal is to change this file (and the others in this directory) as little as possible.   This change needs to be removed.

File: Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/data/RepeatedDynamicDataType.java [hunk 4]
```diff
@@ -29,7 +29,7 @@
  * 
  * Base abstract data type for a Dynamic structure data type that contains
  * some number of repeated data types.  After each data type, including the header
- * there is a terminator value which specifies wether there are any more data structures
+ * there is a terminator value which specifies weather there are any more data structures
```
Comment (dragonmacher) [inline Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/data/RepeatedDynamicDataType.java]: This should be `whether`

File: GPL/DemanglerGnu/src/demangler_gnu/c/cp-demangle.c (hunk 2 above)
Comment (io12) [reply GPL/DemanglerGnu/src/demangler_gnu/c/cp-demangle.c]: Fixed

File: GPL/DemanglerGnu/src/demangler_gnu/c/cplus-dem.c (hunk 3 above)
Comment (io12) [reply GPL/DemanglerGnu/src/demangler_gnu/c/cplus-dem.c]: Fixed

File: Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/data/RepeatedDynamicDataType.java (hunk 4 above)
Comment (io12) [reply Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/data/RepeatedDynamicDataType.java]: Fixed

**Pull request closed without merging**